docs/Using Fleet/Supported-host-operating-systems.md

```diff
@@ -14,23 +14,24 @@ While Fleet may still function partially or fully with OS versions older than th
 ## Some notes on compatibility
 
 ### Tables
+
 Not all osquery tables are available for every OS. Please check out the [osquery schema](https://fleetdm.com/tables) for detailed information. 
 
 If a table is not available for your host, Fleet will generally handle things behind the scenes for you. 
 
-### M1 Macs
-Fleet's agent (fleetd) generated for MacOS by `fleetctl package` does not include native support for M1 Macs. Some values returned may reflect the information returned by Rosetta rather than the system. For example, a CPU will show up as `i486`. 
+### Macs with Apple silicon
+
+Fleet's agent (fleetd) generated for macOS by `fleetctl package` does not include native support for Macs with Apple silicon. Some values returned may reflect the information returned by Rosetta rather than the system. For example, a CPU will show up as `i486`. 
```

Comment:
@lucasmrod is this still the case?

Didn't we add support for arm?

Reply:
It is not the case anymore. fleetd for macOS has components that are universal binaries (run on both Intel and Apple silicon).

Reply:
Sounds good! I removed it. Thanks.